gdc_ng_models/models/genomic_data.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import Boolean, Column, Enum, Index, Text
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+GENOMIC_DATA_TYPE_ENUM = Enum("ssm", "cnv", "case", name="genomic_data_data_type")
+
+
+class GenomicData(Base):
+
+    __tablename__ = "genomic_data"
+    __table_args__ = (
+        Index("genomic_data_project_id_idx", "program_name", "project_code"),
+    )
+
+    def __repr__(self):
+        return "<GenomicData(project_id={}-{}, data_type={}, controlled={}, open='{}')>".format(
+            self.program_name,
+            self.project_code,
+            self.data_type,
+            self.is_controlled,
+            self.is_open,
+        )
+
+    program_name = Column(Text, primary_key=True, nullable=False)
+    project_code = Column(Text, primary_key=True, nullable=False)
+    data_type = Column(GENOMIC_DATA_TYPE_ENUM, nullable=False)
```

Comment:
If we're not giving this table a surrogate key (I don't think it needs a surrogate key), then the `data_type` will also need to be included in the primary key, since some projects will have all data types released.

Reply:
given what @kulgan has mentioned, we might instead define a custom field validator https://docs.sqlalchemy.org/en/13/orm/mapped_attributes.html. The actual `Column` in this case will be of type `Text`